### DISCUSSION TITLE
Accept txt also allow csv

### DISCUSSION THREAD

**Discussion by ajouve** [react-dropzone/react-dropzone#1348] [Q&A] [answered]
Hi,
Not sure if this is a bug or a bad configuration on my side but 

```
useDropzone({accept: {"text/plain": [".txt"]})
```

Allow me to upload `.csv` files

**Comment by rolandjitsu** [comment]
@ajouve does https://react-dropzone.js.org/#section-accepting-specific-file-types help?

**Comment by Chandan-Choubey** [comment]
How can i configure it to accept only csv or xls file??

**Reply by rolandjitsu** [reply, marked as answer]
Please see https://react-dropzone.js.org/#section-accepting-specific-file-types . Is that what you're looking for?

**Reply by Chandan-Choubey** [reply]
Yes It's working now
